control.c

```diff
@@ -173,6 +180,22 @@ void reset_motion(void)
 	reset_control_all();
 }
 
+/**
+ * @brief Return the current motor driver voltage for the left motor.
```

Comment:
- For the description it could seem that you are reading the voltage value from the software driver. The same for right voltage.
- Include motor on the name of the function?

Reply:
9da0495d08db6866d34e6e00efd3dbbd2ffe67f6